guide/05-working-with-the-spatially-enabled-dataframe/part2_data_io_reading_data.ipynb

```diff
@@ -0,0 +1,5229 @@
+{
```

Comment:
<p>A <code>SeDF</code> can be easily created from the data in <a href="https://arrow.apache.org/docs/python/feather.html" target="_blank">feather</a> format using the <code><a href="https://developers.arcgis.com/python/api-reference/arcgis.features.toc.html?highlight=from_feather#arcgis.features.GeoAccessor.from_feather" target="_blank">from_feather()</a></code> method. The method's default <em>SHAPE</em> is the <code>spatial_column</code> for geo-spatial information, but any other column with spatial information can be specified.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='132'/>